code/game/gamemodes/cult/cult.dm

```diff
@@ -1,6 +1,7 @@
 //This file was auto-corrected by findeclaration.exe on 25.5.2012 20:42:31
 
 /datum/game_mode
+	var/datum/cult_info/cult = pick(subtypesof(/datum/cult))

```

Comment:
subtypesof(/datum/cult_info)

Should probably also include the base type, since you didn't define the cult of narsie as a subtype of the datum

Can do that with typesof(/datum/cult_info)